includes/admin/class-wp-job-manager-admin.php

```diff
@@ -163,8 +178,10 @@ public function admin_enqueue_scripts() {
 			}
 		}
 
-		WP_Job_manager::register_script( 'job_manager_notice_dismiss', 'js/admin/wpjm-notice-dismiss.js', null, true );
-		wp_enqueue_script( 'job_manager_notice_dismiss' );
+		if ( $this->are_promoted_jobs_enabled ) {
+			WP_Job_manager::register_script( 'job_manager_notice_dismiss', 'js/admin/wpjm-notice-dismiss.js', null, true );
+			wp_enqueue_script( 'job_manager_notice_dismiss' );
```

Comment:
This script is used for the other notices too, to hook up the dismiss button that WordPress adds to the WPJM notice system.

Reply:
Good catch! Didn't realize that